transaction-service/transaction-module/src/main/kotlin/com/ajaxsystems/infrastructure/web/controllers/TransactionController.kt

```diff
@@ -15,27 +18,18 @@ import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
-import reactor.core.publisher.Flux
-import ua.com.foxminded.restClient.dto.TransactionDto
-import ua.com.foxminded.restClient.service.CurrencyExchangeService
-import ua.com.foxminded.restClient.service.TransactionService
-import ua.com.foxminded.restClient.utils.PageableTransaction
-import java.time.LocalDateTime
-import java.util.Currency
-import java.util.UUID
+import reactor.core.publisher.Mono
 
 @RestController
 class TransactionController(
-    private val transactionService: TransactionService,
-    private val exchangeService: CurrencyExchangeService
+    private val useCase: FindTransactionsRestApiUseCase,
```

Comment:
don't use use case directly, use inPort interface instead

Reply:
it was mistake